R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -1339,6 +1339,11 @@ test_that("column functions", {
   expect_equal(collect(select(df, bround(df$x, 0)))[[1]][2], 4)
 
   # Test to_json(), from_json()
+  arr <- list(listToStruct(list("name" = "bob")))
+  df <- as.DataFrame(list(listToStruct(list("people" = arr))))
+  j <- collect(select(df, alias(to_json(df$people), "json")))
+  expect_equal(j[order(j$json), ][1], "[{\"name\":\"bob\"}]")
+
```

Comment:
Sure, let me add it if I find a better way (it seems `listToStruct` is even an interval API.. ). Is it okay to use `sql("SELECT array(named_struct('name','bob')) as people")` instead if it works fine?